src/intro.md

```diff
@@ -4,13 +4,41 @@
 [« Previous](index.html) • [TOC](index.html) • [Next »](processWorkflow.html)
 :::
 
-When the JDK was released under the GPLv2 license in May 2007, the need for a
-detailed document for contributors was anticipated. Translating rather
-sophisticated institutional knowledge into a format suitable for non-Oracle
-employees is always a challenge. This document attempts to address the needs of
-developers who wish to participate in the future of the JDK.
+Welcome to the OpenJDK Developers' Guide!
 
-Comments may be sent to [guide-dev (at) openjdk.java.net](mailto:guide-dev-at-openjdk.java.net).
+The OpejJDK project governs the Java programming language and the Java runtime.
+The project was created when the JDK was first released under the GPLv2 license in May
+2007 and since then a community has grown around it to help develop and maintain Java.
+
+In order to work together efficiently clear directions are sometimes needed to
+avoid misconceptions and to align developers' views of terminology and process.
+The OpenJDK community is a fairly pragmatic place. "Do what's right" is most
+often the right course of action. Still, if people do things in the same right way
+everyone's work becomes more transparent and easier for others to follow. For
+this reason most parts of the development process have standard flows that are the
+recommended ways to do things.
+
+The goal of this guide is to answer questions that the developers of the JDK might
+have around development process, tooling, standards, etc. The formal rules and
+processes are described in other documents, like [JEP 1](https://openjdk.java.net/jeps/1)
+for the JDK Enhancement-Proposal & Roadmap Process, or [JEP 3](https://openjdk.java.net/jeps/3)
+for the JDK Release Process. This guide is meant to be a complement with tutorials
+and examples for how to follow these rules and how to work together with the rest
+of the OpenJDK community.
+
+There are many common use cases that aren't detailed in the formal process. This
+guide contains the defacto standard for how to work in such cases.
+
+## Maintenance
```

Comment:
A "Maintenance" heading in the intro sounds like maintenance on OpenJDK itself.

The maintenance on the guide itself is rather un-interesting. Maybe you should add a trailing chapter "About this Guide" or something like that, where you can put stuff like that.

Reply:
Agreed. A new chapter added for the about stuff.